### ISSUE TITLE
no updates?

### ISSUE THREAD

**Issue by ssm84** [james-fry/home-assistant-mdi#5]
it has never updated tto new mdi...

**Comment by james-fry** [comment]
Yeah I know. But I don't have much time to support this ongoing. I'd welcome a pr with updates.

**Comment by morgapa** [comment]
with this latest update the page has stopped loading

**Comment by ColtonYYZ** [comment]
I just use this website:  https://pictogrammers.com/library/mdi/.  Is there any way of pulling the images from there into your database?

They're on Github too:  https://github.com/Pictogrammers/pictogrammers.com
